sql-statements/sql-statement-create-table.md

```diff
@@ -215,10 +231,14 @@ The `table_option` currently only supports `AUTO_INCREMENT`, `SHARD_ROW_ID_BITS`
 | `AUTO_INCREMENT` | The initial value of the increment field | `AUTO_INCREMENT` = 5 |
 |`SHARD_ROW_ID_BITS`| To set the number of bits for the implicit `_tidb_rowid` shards |`SHARD_ROW_ID_BITS` = 4|
 |`PRE_SPLIT_REGIONS`| To pre-split `2^(PRE_SPLIT_REGIONS)` Regions when creating a table |`PRE_SPLIT_REGIONS` = 4|
+|`AUTO_ID_CACHE`| To set the auto ID cache size in a TiDB instance. By default, TiDB automatically resize it according to allocation speed |`AUTO_ID_CACHE` = 200|
+|`AUTO_RANDOM_BASE`| To set the initial incremental part value of auto_random. This can be consider as an internal interface. For users, please ignore |`AUTO_RANDOM_BASE` = 0|
```

Comment:
```suggestion
|`AUTO_RANDOM_BASE`| To set the initial auto-increment part value of the `auto_random` attribute. This parameter can be considered as a part of the internal interface. Users can ignore this parameter. 
 |`AUTO_RANDOM_BASE` = 0|
```

Reply:
I think "auto-increment" here and the keyword "auto_increment" can confuse users.

Reply:
Comment updated, PTAL.